OpenRA.Mods.Common/AI/HackyAI.cs

```diff
@@ -245,6 +229,7 @@ static object LoadDecisions(MiniYaml yaml)
 
 	public sealed class HackyAI : ITick, IBot, INotifyDamage
 	{
+		// DEPRECATED: Modules should define a MersenneTwister themselves or use World.LocalRandom.
```

Comment:
I don't see a usecase for modules to define their own RNG, so better to remove this IMO.